### ISSUE TITLE
tour: The phrase `The type *T is a pointer to a T value` was not intuitive for me.

### ISSUE THREAD

**Issue by jcnars** [golang/tour#1561]
Context: https://go.dev/tour/moretypes/1

Change the title above to describe your issue and add your feedback here, including code if necessary

By:
> The type *T is a pointer to a T value.

My mind constructed it to mean: *T is a pointer to a T value...
When in fact &T is a pointer to a T.

The text following it explains stuff, but still the first reading didn't sit well. Maybe it's just me and you can ignore this.

**Comment by jcnars** [comment]
Ah !

I see where I got it mixed up.

The `*T` (as in `*int`) is simply a type that represents a pointer to _base type_ `T` (`int`).
There is also this `*t` which represents the value at address/pointer `t`.

This was beautifully explained in this video: https://youtu.be/sTFJtxJXkaY?si=K2OTwhlDLDpfU5Ro

I wonder if the tour.go.dev pointer page be slightly modified with this aha moment.

**Comment by ALTree** [comment]
Yeah it's talking about the _type_ denoted by the symbol `*T`. The word "type" here is the focus. Closing here, I think the wording is fine.
